README.md

```diff
@@ -41,4 +41,11 @@ bridge](http://vimdoc.sourceforge.net/htmldoc/if_pyth.html#python-vim)):
 [1, 2, 3]
 ```
 
+You can embed neovim into your python application instead of binding to a running neovim instance.
+
+```python
+>>> from neovim import attach
+>>> nvim = neovim.attach('child', argv=["/bin/env", "nvim", "--embed"])
```

Comment:
For parallel form with the above example, leave out the `neovim.`. Simply `attach(...)`.